book/src/ci/github.md

```diff
@@ -157,6 +157,51 @@ By default cargo-dist breaks build tasks onto more machines than strictly necess
 
 
 
+### Checking what your build linked against
+
+> since 0.4.0
+
+Although most Rust builds are statically linked and contain their own Rust dependencies, some crates will end up dynamically linking against system libraries. It's useful to know what your software picked up&mdash;sometimes this will help you catch things you may not have intended, like dynamically linking to OpenSSL, or allow you to check for package manager-provided libraries your users will need to have installed in order to be able to run your software.
+
+cargo-dist provides a linkage report during your CI build in order to allow you to check for this. For macOS and Linux, it's able to categorize the targets it linked against to help you gauge whether or not it's likely to cause problems for your users. To view this, check the detailed view of your CI build and consult the "Linkage report" step from the `upload-local artifacts` jobs.
+
+The report is divided into categories to help you make sense of where these libraries are from and what it might mean for your users. These categories are:
+
+* System: Libraries that come with your operating system. These are usually safe to link against, but you should make sure that they come from a package your users will have. On Linux, these packages are all provided by the system's package manager, and the linkage report includes information about which package includes one of these libraries. On macOS, these packages are shipped with the operating system and not managed by a package manager.
```

Comment:
> Libraries that come with your operating system. These are usually safe to link against, but you should make sure that they come from a package your users will have.

Hmm, this seems kinda-incorrect (we want those things in system-dependencies, right?) and unactionable (how would I know if my users will have it?)